src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -296,6 +300,16 @@ const WorkspaceDashboard = _.flow(
     setTagsList(await Ajax().Workspaces.workspace(namespace, name).deleteTag(tag))
   })
 
+  const loadSasToken = _.flow(
+    withErrorReporting('Error loading SAS token'),
+    Utils.withBusyState(setIsFetchingSasToken)
+  )(async workspaceId => {
+    const response = await Ajax().Workspaces.resources(workspaceId)
+    const container = _.find(resource => resource.metadata.resourceType === 'AZURE_STORAGE_CONTAINER', response.resources)
+    const sasToken = await Ajax().Workspaces.sasToken(workspaceId, container.metadata.resourceId)
```

Comment:
I would consider putting all the logic for getting the sasToken into the `sasToken` method (fetching the resources, finding the AZURE_STORAGE_CONTAINER). At least, that's what I decided to do for `AzureStorage.details`.